test/test_item_model.py

```diff
@@ -635,7 +635,7 @@ def test_insert_after_sort(self):
         data0 = self._data( 0, 1, self.item_model )
         data1 = self._data( 1, 1, self.item_model )
         self.assertTrue( data1 > data0 )
-        self.item_model.sort( 1, Qt.SortOrder.DescendingOrder )
+        self.item_model.sort( 1, Qt.SortOrder.DescendingOrder.value )
```

Comment:
@tim-vdm : ik vraag me hier af wat er juist moet meegegeven worden als laatste argument aan de sort functie : 

Qt.SortOrder.DescendingOrder.value of Qt.SortOrder.DescendingOrder

door nu te veronderstellen dat het de value is die w meegegeven werkt het sorteren in de nieuwe QML table view ... maar of dat nu just is ??

Reply:
Ik denk dat het probleem in camelot/view/crud_action.py zit (lijn 513):

        model_context.proxy.sort(field_name, self.order!=Qt.SortOrder.AscendingOrder)

Daar moeten we denk ik Qt.SortOrder.AscendingOrder of Qt.SortOrder.DescendingOrder meegeven ipv een bool